packages/core/src/Stryker.ts

```diff
@@ -101,14 +104,42 @@ export default class Stryker {
       } finally {
         // `injector.dispose` calls `dispose` on all created instances
         // Namely the `SandboxPool`, `MutantTranspileScheduler` and `ChildProcessProxy` instances
-        await statisticsProcess.sendStatistics();
         await mutationTestProcessInjector.dispose();
         await LogConfigurator.shutdown();
       }
     }
     return Promise.resolve([]);
   }
 
+  private async collectStatistics(mutantResults: MutantResult[]) {
+    try {
+      let config = readConfig(new ConfigReader(this.options, this.log));
+      const statisticsProcess = this.injector
+        .provideValue(coreTokens.httpClient, new HttpClient('httpClient'))
```

Comment:
Ik denk nog steeds dat je deze injectie config beter in de 'main' injector kan configureren. Het is de verantwoordelijkheid van de injector om injectie-spul op te zetten, niet van deze functie